### DISCUSSION TITLE
Can't leave osu when it's running

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#15163] [Q&A] [answered]
I'm on mac.

I'm so used to being able to full screen games and then swipe off them (three fingers or ctrl arrow key) to another space and continue using mac while the game remains full-screened in the background.

Not being able to do this is very restrictive and makes it unpleasant to use osu. Is there a way to leave it while keeipng it open?

**Comment by integerrr** [comment, marked as answer]
[#13351](https://github.com/ppy/osu/discussions/13351#discussioncomment-831131) it doesnt seem like this can be fixed on osu!'s side

**Reply by peppy** [reply]
This is how the OS works, correct. @theonlygusti what you want is "borderless" mode, which uses the style of full-screen you are expecting.

**Reply by integerrr** [reply]
Would borderless increase input latency?

**Reply by peppy** [reply]
Yes. But that said, exclusive fullscreen probably won't be possible in a future macOS release so the lower latency is a limited time special.

**Reply by integerrr** [reply]
And I thought I can finally stick to 1 OS, oh well

**Reply by ghost** [reply]
Why is exclusive fullscreen lower latency?

**Reply by ghost** [reply]
Starting osu on top of a natively-fullscreened app totally breaks macOS. The natively-fullscreened app remains visible, osu not visible at all, but you cannot interact with anything on the screen (even if you close the lid and reopen it you can't log back in) until you wait for the menu to have full loaded (best guess) and hold down ESC to quit osu

**Reply by ghost** [reply]
Windowed turned out to be the best layout for me. Borderless rendered osu tiny in the bottom left corner.
